modules/microshift-gitops-debug.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// microshift_running_apps/microshift-gitops.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-gitops-debug_{context}"]
+= Debugging GitOps on {microshift-short}
+
+There are two ways you debug GitOps applications in your {microshift-short} instance: Using sos reports 
+
+.Prerequisites 
+
+* You must have the `sos` package installed 
+
+.Procedure
+
+. Using sos reports to debug GitOps.
+
+.. Log in to the host as a root user. 
+.. Perform the debug report creation procedure by running the following command:
++
+[source,terminal]
+----
+microshift-sos-report
+----
++
+.Example output 
+[source,terminal]
+----
+sosreport (version 4.5.1)
+
+This command will collect diagnostic and configuration information from
+this Red Hat Enterprise Linux system and installed applications.
+
+An archive containing the collected information will be generated in
+/var/tmp/sos.o0sznf_8 and may be provided to a Red Hat support
+representative.
+
+Any information provided to Red Hat will be treated in accordance with
+the published support policies at:
+
+        Distribution Website : https://www.redhat.com/
+        Commercial Support   : https://www.access.redhat.com/
+
+The generated archive may contain data considered sensitive and its
+content should be reviewed by the originating organization before being
+passed to any third party.
+
+No changes will be made to system configuration.
+
+
+ Setting up archive ...
+ Setting up plugins ...
+ Running plugins. Please wait ...
+
+  Starting 1/2   microshift      [Running: microshift]
+  Starting 2/2   microshift_ovn  [Running: microshift microshift_ovn]
+  Finishing plugins              [Running: microshift]
+
+  Finished running plugins
+
+Found 1 total reports to obfuscate, processing up to 4 concurrently
+
+sosreport-microshift-rhel9-2023-03-31-axjbyxw :    Beginning obfuscation...
+sosreport-microshift-rhel9-2023-03-31-axjbyxw :    Obfuscation completed
+
+Successfully obfuscated 1 report(s)
+
+Creating compressed archive...
+
+A mapping of obfuscated elements is available at
+	/var/tmp/sosreport-microshift-rhel9-2023-03-31-axjbyxw-private_map
+
+Your sosreport has been generated and saved in:
+	/var/tmp/sosreport-microshift-rhel9-2023-03-31-axjbyxw-obfuscated.tar.xz
+
+ Size	444.14KiB
+ Owner	root
+ sha256	922e5ff2db25014585b7c6c749d2c44c8492756d619df5e9838ce863f83d4269
+
+Please send this file to your support representative.
+----
+
+. Using the `oc adm inspect` command to debug on GitOps.
+.. Run the `oc adm inspect` command when in the GitOps namespace 
++
+[source,terminal]
+----
+oc adm inspect applications ns/openshift-gitops
+----
++
+.Example output 
+[source,terminal]
```

Comment:
@vwalek Hello! Do you know how this might look?

Reply:
@kelbrown20 Is the command correct ?

`[redhat@microshift-dev .kube]$ oc adm inspect applications ns/openshift-gitops
error: there is no need to specify a resource type as a separate argument when passing arguments in resource/name form (e.g. 'oc get resource/<resource_name>' instead of 'oc get resource resource/<resource_name>'
`

Reply:
hm, yeah the combination `... applications ns/openshift-gitops` will not work. I see what you want to collect, but not sure what should be the command.  Let me check

Reply:
Hey @kelbrown20 - we can just put `oc adm inspect ns/openshift-gitops` as we don't want to collect information about applications by default. We can always pull that additionally from the cluster with `oc get applications -o yaml  -A`.

Reply:
The example could be:
```
# oc adm inspect ns/openshift-gitops
Gathering data for ns/openshift-gitops...
W0501 20:34:35.978508   57625 util.go:118] the server doesn't have a resource type egressfirewalls, skipping the inspection
W0501 20:34:35.980881   57625 util.go:118] the server doesn't have a resource type egressqoses, skipping the inspection
W0501 20:34:36.040664   57625 util.go:118] the server doesn't have a resource type servicemonitors, skipping the inspection
Wrote inspect data to inspect.local.2673575938140296280.
```

Let me know if this is ok.